frontend_lib/src/component/DisplayState/DisplayState.jsx

```diff
@@ -15,6 +15,7 @@ const DisplayState = props => (
       <button
         className='displaystate__btn'
         onClick={props.onClickBtn}
+        title={props.tooltip}
```

Comment:
I think a html "title" won't be visible enough for user.

It can be in a confirmation popup